CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.0] - 2021-12-17
+
+### Added
+
+* Support for [stateless IRMA server using Redis](https://irma.app/docs/next/stateless)
```

Comment:
could we use the (yet non-exisiting) link to doc version 0.9.0 here instead of `next`?